sysadmin/templates/secrets.json.j2

```diff
@@ -0,0 +1,42 @@
+{
+    "FILENAME": "secrets.json",
+    "DEBUG": 0,
+    "SECRET_KEY": "{{SECRET_KEYS['SECRET_KEY']}}",
+    "DATABASES_HOST": "{{SECRET_KEYS['DATABASES_HOST']}}",
+    "DATABASES_NAME": "{{SECRET_KEYS['DATABASES_NAME']}}",
+    "DATABASES_USER": "{{SECRET_KEYS['DATABASES_USER']}}",
+    "DATABASES_PASSWORD": "{{SECRET_KEYS['DATABASES_PASSWORD']}}",
+    "SOLR_HOST": "{{SECRET_KEYS['SOLR_HOST']}}",
+    "SOLR_PORT": "{{SECRET_KEYS['SOLR_PORT']}}",
+    "SOLR_COLLECTION": "{{SECRET_KEYS['SOLR_COLLECTION']}}",
+    "STATIC_ROOT": "{{oc_install_dir}}/static",
+    "GEOIP_PATH": "{{geoip_path}}",
+    "FILE_CACHE_PATH": "{{oc_install_dir}}/cache/file-cache",
```

Comment:
Possible enhancement: because there is more than one place where a file cache path is derived as `"{{oc_install_dir}}/cache/file-cache"`, it might be helpful to compute that variable in one place (such as in the ansible playboo) and set to an ansible variable like `file_cache_path` that then gets inserted in secrets.json.j2